gammapy/time/models.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import numpy as np
+from ..utils.modeling import Parameter, ParameterList
+
+__all__ = [
+    'PhaseCurve',
+]
+
+log = logging.getLogger(__name__)
+
+
+class PhaseCurve(object):
+    """PhaseCurve class.
+    
+    This is to calculate phase of a periodic system and to provide the 
+    normalization factor for the phase using circular interpolation. 
+    The required data for interpolation is given as an astropy table
+
+    # TODO: 1. Checking table values if the phases are in increasing order
```

Comment:
Can you resolve the TODOs? Or explain what the problem is? Is it needed?

Reply:
Actually, we don't need these TODOs. This was for some checks. Now I feel that it is not correct to put these checks. Hence, I remove TODOs from the docstring